pandas/io/html.py

```diff
@@ -1143,6 +1144,13 @@ def read_html(
 
         .. versionadded:: 1.5.0
 
+    use_nullable_dtypes : bool = False
+        Whether to use nullable dtypes as default when reading data. If
+        set to True, nullable dtypes are used for all dtypes that have a nullable
+        implementation, even if no nulls are present.
```

Comment:
Could you add the additional paragraph of `mode.dtype_backend` being available that other docstrings have? (Should start with `The nullable dtype implementation`)

Reply:
Thx, added